components/brave_rewards/browser/rewards_service.h

```diff
@@ -104,7 +105,7 @@ class RewardsService : public KeyedService {
   virtual void GetReconcileStamp(
       const GetReconcileStampCallback& callback) = 0;
   virtual void GetAddresses(const GetAddressesCallback& callback) = 0;
-  virtual void SetRewardsMainEnabled(bool enabled) const = 0;
+  virtual void SetRewardsMainEnabled(bool enabled) = 0;
```

Comment:
not sure why this was ever `const` in the first place since it's a setter

Reply:
This previously only modified a value in the state and didn't modify any values of the object. It doesn't look like `const` would work now since more stuff is happening. 

@bridiver Is the previous use of `const` not correct?

Reply:
@jasonrsadler the compiler won't complain about it, but I think it violates the principle because this class is effectively acting as a proxy to the class that is being modified.